internal/provider/resource_hvn.go

```diff
@@ -89,6 +89,11 @@ func resourceHvn() *schema.Resource {
 				Type:        schema.TypeString,
 				Computed:    true,
 			},
+			"provider_account_id": {
```

Comment:
Let's save this for a follow-up, but I think this is a good point to introduce the pattern for returning cloud-specific outputs.  I'm thinking we'll have an output with the top-level name `aws`, which contains `account_id`, similar to this [vpc_config object](https://github.com/hashicorp/terraform-provider-aws/blob/main/aws/resource_aws_eks_cluster.go#L165-L214) in the the AWS provider's eks_cluster resource. It's a bit of a hack, but it sounds like it's the best we've got while the provider SDK team works out a better solution for input/output objects.

Reply:
Yea, I think this would warrant some more discussion, so my preference would be to include that in the general topic around cloud-specific values in our network resources. Definitely agree that this field falls into that category.